identity/app/conf/IdentityConfiguration.scala

```diff
@@ -20,4 +21,17 @@ class IdentityConfiguration extends GuardianConfiguration with SafeLogging {
     }
   }
 
+  // Userhelp - 'Guardian Admin' ET Account integrated with subscriptions.dev@guardian.co.uk AppCentre
+  object exactTargetUserhelp {
```

Comment:
I am with @adamnfish on this. I don't think the responsibility of managing user personal data should leave in Frontend. @markjamesbutler has probably an opinion about that